tests/python/contrib/test_vitis_ai_codegen.py

```diff
@@ -0,0 +1,203 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=no-else-return, unidiomatic-typecheck, invalid-name, W0611
+"""Vitis-AI codegen tests."""
+
+import numpy as np
+
+import tvm
+from tvm import relay
+from tvm.relay import transform
+from tvm.relay.op.contrib.vitis_ai import annotation
+from tvm.contrib.target import vitis_ai
+
+import pyxir
+import pyxir.contrib.target.DPUCADX8G
```

Comment:
It might be a problem to directly import Vitis-AI dependencies, because most developers may not have this environment. Please make all Vitis-AI specific imports lazy, and safely skip all unit tests if the required environment is not ready.

Reply:
Yes, indeed, we will be adding a pytest importorskip at the top of these test files.